modules/terraform/terraform.go

```diff
@@ -1,2 +1,13 @@
 // Package terraform allows to interact with Terraform.
 package terraform
+
+// https://www.terraform.io/docs/commands/plan.html#detailed-exitcode
+
+// TerraformPlanChangesPresentExitCode is the exit code returned by terraform plan detailed exitcode when changes are present
+const TerraformPlanChangesPresentExitCode = 2
+
+// TerraformPlanNoChangeExitCode is the exit code returned by terraform plan detailed exitcode when no changes are present
+const TerraformPlanNoChangeExitCode = 0
+
+// TerraformPlanExceptionExitCode is the exit code returned by terraform plan detailed exitcode when some exception occurs
+const TerraformPlanExceptionExitCode = -1
```

Comment:
I think exit codes have to be between 0 and 255 in *nix. So `1` is probably a better, more canonical value here.

Reply:
switched to `DefaultErrorExitCode` with value 1